### PR TITLE
Integrate ALP and WAF changes in Cloud docs

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Comment by girishshankaran** [comment]
@tmjd Can you please review this one?

**Comment by ctauchen** [comment]
LGTM, plus OK from SME

File: calico-cloud/threat/web-application-firewall.mdx [hunk 1]
```diff
@@ -65,123 +64,111 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
...
-| Block  | Despite the name, this **will not block or drop the request**. ModSecurity will return detection=0 in this case and Calico will **not** log the event in Elasticsearch. | No          |
-| Deny   | Denies HTTP traffic as ModSecurity will return detection=1 and will log the event in Elasticsearch.                                                                     | Yes         |
-| Drop   | Denies HTTP traffic as ModSecurity will return detection=1 and will log the event in Elasticsearch.                                                                     | Yes         |
+#### Kibana
 
-#### Add or edit a rule set
+To view WAF events In Kibana:
 
-Create a directory, and download the core rules set files that you want to use, for example:
+1. Create the `tigera_secure_ee_waf*` index pattern. For help with index patterns, see [Kibana index pattern](https://docs.tigera.io/calico-cloud/visibility/kibana).
```
Comment (tmjd) [inline calico-cloud/threat/web-application-firewall.mdx:122]: Should this URL should have `docs.tigera.io` (and `calico-cloud`) here? I would have thought those should be {{something}} or a relative link.

File: calico-cloud/threat/web-application-firewall.mdx [hunk 2]
```diff
@@ -215,47 +204,102 @@ root@waf-attacker-1234:/ curl http://<host>//test/artists.php?artist=0+div+1+uni
...
+
+To disable WAF on a service, use the Actions menu on the WAF board, or use the following command:
+
+```
+kubectl annotate svc <service-name> -n <service-namespace> projectcalico.org/l7-logging-
+```
+
+#### Disable WAF feature
+
+To disable WAF, update the [ApplicationLayer](https://docs.tigera.io/reference/installation/api#operator.tigera.io/v1.ApplicationLayer) resource to include the `webApplicationFirewall` field, and ensure it is set to `Disabled`.
```
Comment (tmjd) [inline calico-cloud/threat/web-application-firewall.mdx:294]: Same as my other comment, seems like this shouldn't hard code `docs.tigera.io`. Also shouldn't this include `calico-cloud` in the path?

File: calico-cloud/threat/web-application-firewall.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico-cloud/threat/web-application-firewall.mdx:122]: Can I put something like this? [Kibana index pattern] (../visibility/kibana.mdx)?

Comment (doucol) [reply calico-cloud/threat/web-application-firewall.mdx:122]: yes, I think that should work - try it and let us know!

File: calico-cloud/threat/web-application-firewall.mdx (hunk 2 above)
Comment (doucol) [reply calico-cloud/threat/web-application-firewall.mdx:294]: yes, absolutely - we should never be putting hard coded references like that

Comment (doucol) [reply calico-cloud/threat/web-application-firewall.mdx:294]: {{baseUrl}}

**Comment by girishshankaran** [comment]
@tmjd The suggested changes are addressed in this [PR](https://github.com/tigera/docs/pull/585).

File: calico-cloud/network-policy/application-layer-policies/alp.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+### Enforce application layer policies for ingress traffic
+
+You can restrict ingress traffic using HTTP match criteria using Network policy or Global network policy. 
+For a list of all HTTP match parameters, see [Network policy](/reference/resources/networkpolicy.mdx) and [Global network policy](/reference/resources/globalnetworkpolicy.mdx).
+
+In the following example, the trading app is allowed ingress traffic only for HTTP GET requests that match the exact path /projects/calico, or that begins with the prefix, /users.
+
+```
```
Comment (bmckercher123) [inline calico-cloud/network-policy/application-layer-policies/alp.mdx:45]: YAML

**Pull request merged**